examples/06-dc/plot_dc_schlumberger_array.py

```diff
@@ -266,3 +266,10 @@
 ax[1].set_xlabel('Conductivity (S/m)', fontsize=25)
 ax[1].set_ylabel('Depth (m)', fontsize=25)
 ax[1].set_title('(b)', fontsize=25)
+
+###############################################################################
```

Comment:
display the versions in the notebook-styled examples

Reply:
You say "notebook-styled" examples; wouldn't you want to add then `versions('HTML')` instead of `versions()`?

Reply:
Never mind. it is a pure Python file I see, so it was your comment that mislead me.

Reply:
We need it to run both as a stand-alone python script and as a notebook, so I figured sticking with the defaults would be simpler